.github/workflows/reuse.linux.yml

```diff
@@ -346,12 +346,12 @@ jobs:
 
   test_ee_standalone:
     runs-on: [self-hosted, X64, Linux, 4c8g]
-    if: github.repository == github.event.pull_request.head.repo.full_name
+    if: github.event_name != 'pull_request' || github.repository == github.event.pull_request.head.repo.full_name
```

Comment:
I like to use `github.event_name == 'push' || !github.event.pull_request.head.repo.fork`, maybe also works well here?